lib/addons/fluxcd/index.ts

```diff
@@ -61,7 +39,7 @@ export interface FluxCDAddOnProps extends HelmAddOnUserProps {
 
   /** 
   * Flux Kustomization Target Namespace.
-  * Note: when set, this parameter will override all the objects that are part of the Kustomization, please see https://fluxcd.io/flux/components/kustomize/kustomization/#target-namespace */
+  * Default `default` */
   fluxTargetNamespace?: string;

```

Comment:
is this namespace ever different for these apps?  my assumption is that this is the namespace where the GitRepository CR will be created, please confirm. If my assumption is correct, this namespace is going to be the same for all of them. 
If it is something else let's discuss.

Reply:
The target namespace is where we want the kustomizations to be deployed.  This can be specified per kustomization, whereas the gitrespositories are deployed in the same namespace that flux is deployed

Reply:
The name doesn't convey it, should be fixed, e.g. kustomizationNamespace or kustomizationTargetNamespace. 
If there is a particular set of attributes specific to kustomizations then it would make more sense to have an interface for `Kustomization` with path, namespace, etc.

Reply:
> is this namespace ever different for these apps? my assumption is that this is the namespace where the GitRepository CR will be created, please confirm. If my assumption is correct, this namespace is going to be the same for all of them. If it is something else let's discuss.

I think we might want NOT to make the assumption of a single namespace. E.g. in this case https://github.com/aws-observability/aws-observability-accelerator/blob/main/artifacts/grafana-operator-manifests/eks/infrastructure/amg_grafana-amp-datasource.yaml we use grafana-operator as a namespace, but we might want to add another path, with other manifests related to another operator and so likely another namespace.
I agree on the name. The parameter is finally going to be applied to https://fluxcd.io/flux/components/kustomize/kustomization/#target-namespace and like mentioned elsewhere, it should be optional, to avoid forcing override.